### PR TITLE
Adding FSI and Healthcare themes for Adaptive Form

### PR DESCRIPTION
**Pull request by pankaj-parashar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by suryansh29** [comment]
I can see back and cancel image in assets, they will not be localized.
src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-back.png
src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-cancel.png

and continue and submit in images as well
src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-continue.png
src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-theme2-submit.png

File: src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/.content.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:dam="http://www.day.com/dam/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="dam:Asset">
+    <jcr:content
+        cq:conf=""
+        jcr:lastModified="{Date}2022-09-01T18:48:10.103-07:00"
+        jcr:primaryType="dam:AssetContent"
```
Comment (suryansh29) [inline src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/.content.xml:7]: only assets and renditions files are present in the theme, aren't there any other files in theme?

Comment (pankaj-parashar) [reply src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/.content.xml:7]: This is AF1 theme. Theme definition is at [/fsi/_jcr_content/renditions/theme-json/.content.xml](https://github.com/adobe/aem-project-archetype/pull/1039/files#diff-35d4c4f2704b2965011c960dfa38c389a59c153b3b9f41ca58ff0cc2561bb637) When imported to the system, clientlibrary is generated by a listener.

**Comment by suryansh29** [comment]
For Back and cancel I see PNG images, it will not be localized
src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-back.png

src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/assets/mobile-cancel.png
@pankaj-parashar

**Comment by pankaj-parashar** [comment]
> For Back and cancel I see PNG images, it will not be localized src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/**appId**/fsi/assets/mobile-back.png
> 
> src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/**appId**/fsi/assets/mobile-cancel.png @pankaj-parashar

I see that these images rules are not effective for device renditions.

**Review by suryansh29 — APPROVED** [review]

**Review by suryansh29 — APPROVED** [review]

**Review by rismehta — APPROVED** [review]
LGTM

**Pull request merged**